mlflow/data/spark_delta_utils.py

```diff
@@ -85,8 +87,11 @@ def _try_get_delta_table_latest_version_from_table_name(table_name: str) -> Opti
 
     try:
         spark = SparkSession.builder.getOrCreate()
+        backticked_table_name = ".".join(
+            part if _is_backticked(part) else f"`{part}`" for part in table_name.split(".")
+        )
```

Comment:
Can we make this a function?

Reply:
I'd make that function if there were more names to backtick (say 5).

Reply:
But the same logic is used twice, I think `backtick_delta_table_name` is more helpful than `_is_backticked` method

Reply:
ok, I'll make that funciton.

Reply:
Here's the comparison of how I view this code:

```python
        # It's clear that each part of the table name should be backtick quoted
        backticked_table_name = ".".join(map(_backtick_quote, table_name.split(".")))
        # Does this quote the entire table name or each part?
        backticked_table_name = backtick_delta_table_name(table_name)
```